client/components/chart/bar.jsx

```diff
@@ -29,23 +31,23 @@ module.exports = React.createClass( {
 	},
 
 	buildSections: function() {
-		var value = this.props.data.value,
+		const value = this.props.data.value,
```

Comment:
Optional, but we can [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) a bit:
```javascript
const { active, data, max } = this.props;
const { nestedValue, value } = data;
```